lib/tables.dart

```diff
@@ -13,7 +14,7 @@ import 'utils.dart';
 
 // TODO(devoncarew): fixed position header
 
-class Table<T> extends Object with SetStateMixin, OnAddedToDomMixin {
+class Table<T> extends Object with SetStateMixin {
```

Comment:
I believe this was the only use of `OnAddedToDomMixin`; we should be able to remove that class (and `CoreElementOwner` as well?).

Reply:
Removed. @DanTup feel free to add back OnAddedToDomMixin if you find a case where it is needed. I expect we can manually trigger some dispose calls if there are some lifecycle cases we really care about.